api/TextTrackList.json

```diff
@@ -246,169 +252,13 @@
-          "support": {
-            "chrome": {
-              "version_added": "33"
-            },
-            "chrome_android": {
-              "version_added": "33"
-            },
-            "edge": {
-              "version_added": "18"
-            },
-            "firefox": {
-              "version_added": "31"
-            },
-            "firefox_android": {
-              "version_added": "31"
-            },
-            "ie": {
-              "version_added": false
-            },
-            "opera": {
-              "version_added": "20"
-            },
-            "opera_android": {
-              "version_added": "20"
-            },
-            "safari": {
-              "version_added": "7"
-            },
-            "safari_ios": {
-              "version_added": "7"
-            },
-            "samsunginternet_android": {
-              "version_added": "2.0"
-            },
-            "webview_android": {
-              "version_added": "4.4"
-            }
-          },
-          "status": {
-            "experimental": false,
-            "standard_track": true,
-            "deprecated": false
-          }
-        }
-      },
-      "onremovetrack": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/TextTrackList/onremovetrack",
-          "spec_url": "https://html.spec.whatwg.org/multipage/media.html#handler-tracklist-onremovetrack",
-          "support": {
-            "chrome": {
-              "version_added": "33"
-            },
-            "chrome_android": {
-              "version_added": "33"
-            },
-            "edge": {
-              "version_added": "18"
-            },
-            "firefox": {
-              "version_added": "31"
-            },
-            "firefox_android": {
-              "version_added": "31"
-            },
-            "ie": {
-              "version_added": false
-            },
-            "opera": [
-              {
-                "version_added": "20"
-              },
-              {
-                "version_added": "≤12.1",
-                "version_removed": "15"
-              }
-            ],
-            "opera_android": [
-              {
-                "version_added": "20"
-              },
-              {
-                "version_added": "≤12.1",
-                "version_removed": "14"
-              }
-            ],
-            "safari": {
-              "version_added": "7"
-            },
-            "safari_ios": {
-              "version_added": "7"
-            },
-            "samsunginternet_android": {
-              "version_added": "2.0"
-            },
-            "webview_android": {
-              "version_added": "4.4"
-            }
-          },
-          "status": {
-            "experimental": false,
-            "standard_track": true,
-            "deprecated": false
-          }
-        }
-      },
       "removetrack_event": {
```

Comment:
The event and the property were added at the same time in WebKit and Blink:
https://trac.webkit.org/changeset/148244/webkit
https://chromium.googlesource.com/chromium/src/+/1947d7da50d5bb254bc2baae2609cc9d06d6cfff